saleor/plugins/avatax/plugin.py

```diff
@@ -119,15 +119,6 @@ def _skip_plugin(
         if not self.active:
             return True
 
-        # The previous plugin already calculated taxes so we can skip our logic
```

Comment:
Why did you remove this part of code? 
```
if isinstance(previous_value, TaxedMoney):
    return previous_value.net != previous_value.gross
``` 
Allows us to not calculate taxes twice for the same line.